testing/constraints-3.7.txt

```diff
@@ -19,7 +19,7 @@ opentelemetry-sdk==1.1.0
 pandas==1.1.0
 proto-plus==1.15.0
 protobuf==3.12.0
-pyarrow==3.0.0
+pyarrow>=5.0.0
```

Comment:
generally we test with a specific version here and it should match our minimum in setup.py. Maybe we need another conditional for the pyarrow 5.0+ features?